modules/anti-ghostping/events/messageDelete.js

```diff
@@ -6,6 +6,12 @@ module.exports.run = async function (client, msg) {
     if (!messageWithMentions[msg.id]) return;
     const moduleStrings = client.configurations['anti-ghostping']['config'];
     if (messageWithMentions[msg.id].author.bot) return;
+	if (moduleStrings.ignoredChannels.includes(msg.channel.id)) return;
+    let whitelisted = false;
+    moduleStrings.ignoredRoles.forEach(r => {
+        if (msg.member.roles.cache.get(r)) whitelisted = true;
+    });
+    if (whitelisted) return;
```

Comment:
Code-Readability could be improved massively by using the `.find` / `.filter` function. 
Here's an example: 
```javascript
if (msg.member.roles.cache.find(r => moduleStrings.ignoredRoles.includes(r.id))) return;
```
Please note that I did not test this call and this is purely a suggestion. Feel free to pass feedback or state your opinion.

Reply:
Oh yea thats a lot better, I changed it now.